package.json

```diff
@@ -25,5 +25,8 @@
             "components/supervisor/frontend",
             "charts/"
         ]
+    },
+    "dependencies": {
```

Comment:
should this be here?

Reply:
no, it should be in the package where we have the dependency